imap_processing/cdf/config/imap_codice_l2-hi-sectored_variable_attrs.yaml

```
@@ -0,0 +1,422 @@
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_cno
+
+energy_fe_minus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Minus value
+  FIELDNAM: energy delta minus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_fe
+
+energy_fe_plus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Plus value
+  FIELDNAM: energy delta plus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_fe
+
+energy_h_minus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Minus value
+  FIELDNAM: energy delta minus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_h
+
+energy_h_plus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Plus value
+  FIELDNAM: energy delta plus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_h
+
+energy_he3he4_minus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Minus value
+  FIELDNAM: energy delta minus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_he3he4
+
+energy_he3he4_plus:
+  VAR_TYPE: support_data
+  CATDESC: Energy Table Plus value
+  FIELDNAM: energy delta plus
+  LABLAXIS: Energy
+  SCALETYP: log
+  UNITS: MeV/nuc
+  FORMAT: '%f'
+  FILLVAL: *real_fillval
+  VALIDMAX: 200.0
+  VALIDMIN: 0.05000000074505806
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:EnergyPerCharge,Qualifier:Uncertainty
+  DEPEND_1: energy_he3he4
+
+# spin angles:
+spin_angles:
+  VAR_TYPE: support_data
+  CATDESC: Spin Angle
+  FIELDNAM: Spin Angle
+  SCALETYP: linear
+  UNITS: degrees
+  FILLVAL: *real_fillval
+  VALIDMAX: 360.0
+  VALIDMIN: 0.0
+  FORMAT: '%f'
+  DICT_KEY: SPASE>Particle>ParticleType:Ion,ParticleQuantity:ArrivalDirection,Qualifier:DirectionAngle.AzimuthAngle
+  DEPEND_1: spin_sector
+  DEPEND_2: elevation_angle
+  LABL_PTR_1: spin_sector_label
+  LABL_PTR_2: elevation_angle_label
```

```suggestion
  DEPEND_1: spin_sector
  DEPEND_2: inst_az
  LABL_PTR_1: spin_sector_label
  LABL_PTR_2: inst_az_label
```
spin_angles = 2D array of (spin_sector, inst_az)

---

dont' make changes per discussion for this and above